src/patch_parser.py

```diff
@@ -473,14 +504,14 @@ def _parse_patch_file_removed_chunk(
 
 def _parse_patch_file_chunk(lines: InputSource,
                             gerrit_orig_line: int,
-                            gerrit_new_line: int) -> Tuple[int, int, PatchFileChunkLineMap]:
+                            gerrit_new_line: int, chunks: PatchFileChunkLineMap) -> Tuple[int, int, PatchFileChunkLineMap]:
```

Comment:
we don't seem to be using the newly added chunks parameter here or in `__parse_patch_file_added_chunk()`.
I assume it's a leftover from before when you were considering inspecting the previous hunk to find `deleted_lines()`?